source/cloud/nvidia/brev.md

```diff
@@ -4,41 +4,66 @@ review_priority: "p0"
 
 # NVIDIA Brev
 
-The [NVIDIA Brev](https://brev.dev/) platform provides you with GPU instances on cloud providers including [Amazon Web Services](https://aws.amazon.com/) and [Google Cloud](https://cloud.google.com) with CUDA, Python, Jupyter Lab, all set up.
+The [NVIDIA Brev](https://brev.dev/) platform provides you a one stop menu of available GPU instances across many cloud providers, including [Amazon Web Services](https://aws.amazon.com/) and [Google Cloud](https://cloud.google.com), with CUDA, Python, Jupyter Lab, all set up.
 
-You can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart.
+## Brev Instance Setup 
+There are two ways that you can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart:
+1. Brev GPU Instances - quickly get the GPU, across most clouds, to get your work done.
+2. Brev Launchables - quickly create one-click starting, reusable instances that you customized to your MLOps needs.
 
-## Setting up your instance
+### 1. Setting up your Brev GPU Instance
 
-Navigate to the [Brev console](https://console.brev.dev/) and click on "Create your first instance".
+1. Navigate to the [Brev console](https://console.brev.dev/) and click on "Create your first instance".
 
 ![Screenshot of the "Create your first instance" UI](/_static/images/platforms/brev/brev1.png)
 
-Select "Container Mode".
+2. Select "Container Mode".
 
 ![Screenshot showing "Container Mode" highlighted](/_static/images/platforms/brev/brev2.png)
 
-Attach the "NVIDIA RAPIDS" Container.
+3. Attach the "NVIDIA RAPIDS" Container.
 
 ![Screenshot showing the "NVIDIA RAPIDS" container highlighted](/_static/images/platforms/brev/brev3.png)
 
-Configure your own instance.
+4. Configure your own instance.
 
 ![Screenshot of the "Create your Instance" UI with an NVIDIA T4 GPU highlighted](/_static/images/platforms/brev/brev4.png)
 
-And hit "Deploy".
+5. And hit "Deploy".
 
 ![Screenshot of the instance creation summary screen with the deploy button highlighted](/_static/images/platforms/brev/brev5.png)
 
+### 2. Setting up your Brev Launchable
+1. Click Create Launchable
```

Comment:
Can we link to where people should be clicking this?

Reply:
sure